templates/cluster/vsphere-standalone-cp/templates/vspheremachinetemplate-controlplane.yaml

```diff
@@ -17,8 +17,9 @@ spec:
           networkName: {{ .Values.controlPlane.network }}
           {{- if .Values.ipamEnabled  }}
           addressesFromPools:
-            - {{- toYaml .Values.ipPool.data | nindent 14 }}
+            - {{- toYaml .Values.ipPool.config | nindent 14 }}
           {{- end }}
+          nameservers: {{ .Values.nameservers }}
```

Comment:
The same question as for `templates/cluster/vsphere-standalone-cp/templates/vspheremachinetemplate-worker.yaml`